executor/load_data.go

```diff
@@ -267,7 +267,13 @@ func (e *LoadDataInfo) colsToRow(cols []string) types.DatumRow {
 			e.row[i].SetString("")
```

Comment:
this should be SetNull() too right?

Reply:
Yes, but there may be some corner cases for it.